src/core/database.py

```diff
@@ -0,0 +1,73 @@
+# coding: utf8
+
+__contact__ = "info@hytech-imaging.fr"
+__copyright__ = "Copyright (c) 2021 Hytech Imaging"
+
+import os.path
+
+from PyQt5.QtWidgets import QMessageBox
+from qgis._core import (
+    QgsWkbTypes,
+    QgsFields,
+    QgsVectorFileWriter,
+    QgsCoordinateReferenceSystem,
+    QgsCoordinateTransformContext,
+    QgsFeatureSink,
+    QgsVectorLayer,
+    QgsProject,
+)
+
+
+class SammoDataBase:
+    CONST_DB_NAME = "sammo-boat.gpkg"
+    CONST_LAYER_NAME = "session datas"
+
+    def __init__(self):
+        pass
+
+    def IsDataBaseAvailableInThisDirectory(self, directory):
```

Comment:
camelCase -> `isDataBaseAvailableInThisDirectory`

Reply:
Done for all methods in 5th commit (20ab29a790088aff686754a3616766e69a0b3e25)